frontend/package.json

```diff
@@ -9,7 +9,9 @@
     "react-fontawesome": "1.5.0",
     "react-router-dom": "4.3.1",
     "react-scripts": "2.0.5",
-    "terser": "^3.14.1"
+    "terser": "^3.14.1",
+    "react-bootstrap-table2-paginator": "^2.0.3",
+    "reactstrap": "^7.1.0"
```

Comment:
CircleCI is complaining about a missing `react-bootstrap-table-next` library. Maybe that needs to be npm installed to add it here?

Reply:
Think this still needs to be addressed.